packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -4,6 +4,9 @@ import React from 'react';
 import classNames from 'classnames';
 import uniqueId from 'lodash.uniqueid';
 
+export type ChoiceSize = 'small';
+export type ChoiceType = 'checkbox' | 'radio';
+export type ChoiceValue = number | string;
 export interface ChoiceProps {
```

Comment:
I think there may be a conflict with the use of `ChoiceProps` here. At least in what I'm seeing in Aaron's screenshot, it looks like this actually gets exported from the ChoiceList file instead of Choice. I found a reference to what I think they're using, but not quite sure: https://github.com/CMSgov/design-system/blob/core-2.6.1/packages/design-system/src/types/ChoiceList/ChoiceList.d.ts#L10

Reply:
From what I'm seeing, I think `ChoiceProps` is exported from both files. It is exported here (so it can be imported into `ChoiceList` and also from `ChoiceList` (which is where Aaron's team is importing it from)

Here's where I'm seeing it exported from the original file: https://github.com/CMSgov/design-system/blob/core-2.6.1/packages/design-system/src/types/ChoiceList/Choice.d.ts